clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -369,33 +375,33 @@ public void close(long duration, TimeUnit unit) {
         while (true) {
             if (hardShutdownTimeMs.compareAndSet(prev, newHardShutdownTimeMs)) {
                 if (prev == INVALID_SHUTDOWN_TIME) {
-                    log.debug("{}: initiating close operation.", clientId);
+                    log.debug("initiating close operation.");
```

Comment:
Let's capitalize the log statements for consistency. There are a few of these.

Reply:
Updated the log statements to start with capital letters. Thanks for the review.